styx-scheduler-service/src/main/java/com/spotify/styx/state/StateManager.java

```diff
@@ -64,6 +64,12 @@
   Map<WorkflowInstance, RunState> activeStates();
 
   /**
+   * Get a map of all active {@link WorkflowInstance} states whose issuing {@link Trigger} is of
+   * kind "backfill".
+   */
+  Map<WorkflowInstance, RunState> backfillActiveStates();
```

Comment:
I'm not sure this is the state managers concern? Should it be internal to the scheduler?